test/Microsoft.AspNet.Mvc.WebApiCompatShimTest/ApiControllerActionDiscoveryTest.cs

```diff
@@ -11,7 +11,7 @@
 using Microsoft.AspNet.Mvc.Core;
 using Microsoft.AspNet.Mvc.Filters;
 using Microsoft.AspNet.Mvc.WebApiCompatShim;
-using Microsoft.Framework.Logging;
+using Microsoft.AspNet.Testing.Logging;
```

Comment:
nit: one of the `NullLogger` mentions in this file uses `new NullLogger()` while the reset use `NullLogger.Instance`.  let's be consistent and use `Instance` everywhere (just one case to change).